rpcs3/Emu/Cell/lv2/lv2.cpp

```diff
@@ -1022,6 +1022,8 @@ class ppu_syscall_usage
 
 	void print_stats() noexcept
 	{
+		return;
```

Comment:
probably a remnant of debugging?
usually you can easily avoid adding these random code segments in git.

Reply:
no, this is all to reduce spam in the logs I get from testers